### DISCUSSION TITLE
Welcome XCLogParser to MNF!

### DISCUSSION THREAD

**Discussion by BalestraPatrick** [MobileNativeFoundation/discussions#98] [General]
Hello! 

I'm making this quick announcement in regards to XCLogParser.

XCLogParser was started over 2 years ago by some engineers at Spotify (mainly @ecamacho) in our hack days with the goal of automatically collecting Xcode logs and inspecting our developer experience. A couple months later we decided to make it open-source under the Spotify organization. The project evolved a lot in the following months by adding a few new features with the help of community as well as keeping it up to date with new Xcode releases.

We are happy to have donated the project to the Mobile Native Foundation to allow the community to continue developing and maintaining the project for the years to come. If you'd like to be involved, head over to new project location living in the MNF organization: https://github.com/MobileNativeFoundation/XCLogParser

Thanks everyone! 🙏

**Comment by keith** [comment]
Thanks @BalestraPatrick and everyone at Spotify for making this happen, I think this is a great project to be part of the MNF
